packages/frontend2/src/app/(legacy)/zk-catalog/[project]/page.tsx

```diff
@@ -0,0 +1,59 @@
+import { type Metadata } from 'next'
+import { notFound } from 'next/navigation'
+import { ContentWrapper } from '~/app/_components/content-wrapper'
+import { getVerifiers } from '~/server/features/zk-catalog/get-verifiers'
+import { getDefaultMetadata } from '~/utils/get-default-metadata'
+import { ZK_CATALOG_ASK_FOR_VERIFICATION_LINK } from '../_utils/getZkCatalogView'
+import { projects } from '../_utils/projects'
+import { ZkCatalogProjectPage } from './_components/ZkCatalogProjectPage'
+import { getZkCatalogProjectDetails } from './_utils/getZkCatalogProjectDetails'
+
+interface Props {
+  params: {
+    project: string
+  }
+}
+
+// Do we want to prerender these pages?
```

Comment:
I guess for not we can't because of the database env issue. Let's remove this comment.

Reply:
https://github.com/l2beat/l2beat/pull/4378/commits/40a77e61d089db4b42d884eef2907b2cbce1f1b6